src/Storages/Kafka/StorageKafka2.cpp

```diff
@@ -8,10 +8,10 @@
 #include <IO/EmptyReadBuffer.h>
```

Comment:
Please, remove the changes that is from `clang-format` or similar tools, leave only part that is related to the PR